.circleci/config.yml

```diff
@@ -593,16 +471,25 @@ jobs:
             else
                echo '' > .CACHE-PREFIX
             fi
+      - run:
+          name: Create static visualization js bundle
+          command: yarn build-static-viz
       - run:
           name: 'Check for branch name to bust caches if it is a release branch'
           command: |
             if [[ $CIRCLE_BRANCH == release* || $CIRCLE_BRANCH == master  ]]; then
               echo 'This is a release or master branch; using cache-busting prefix'
                 echo '<< pipeline.id >>' > .CACHE-PREFIX
             fi
-      - run-yarn-command:
-          command-name: Create static visualization js bundle
-          command: build-static-viz
+      - run:
+          name: Make SSL certificates for Mongo available
+          command: >-
+            curl https://raw.githubusercontent.com/metabase/metabase-qa/master/dbs/mongo/certificates/metabase.crt
+            -o /home/circleci/metabase/metabase/test_resources/ssl/mongo/metabase.crt
+            https://raw.githubusercontent.com/metabase/metabase-qa/master/dbs/mongo/certificates/metabase.key
+            -o /home/circleci/metabase/metabase/test_resources/ssl/mongo/metabase.key
+            https://raw.githubusercontent.com/metabase/metabase-qa/master/dbs/mongo/certificates/metaca.crt
+            -o /home/circleci/metabase/metabase/test_resources/ssl/mongo/metaca.crt
```

Comment:
Are you sure about these? That was added to 44, not 43.

Reply:
Yeah.
Removed it in https://github.com/metabase/metabase/pull/23573/commits/2bbd60955d881efe327745bbb9cac6f75eec0afc. It was a bad initial merge conflict resolution on my part.